README.md

```diff
@@ -100,17 +98,13 @@ region = eu-west-2
 
 ## Configure kubectl
 
-To configure kubetcl, you need both [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html). You only need the AWS IAM Authenticator if your installed AWS CLI is older than release 1.16.156. 
-Otherwise you can simply run the below commnand to get the access credentials for your cluster and automatically configure `kubectl`.
+To configure kubetcl, you need both [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html). You only need the AWS IAM Authenticator if your installed AWS CLI is older than release 1.16.156. Otherwise you can simply run the below commnand to get the access credentials for your cluster and automatically configure `kubectl`.
 
 ```shell
 $ aws eks --region $(terraform output region) update-kubeconfig --name $(terraform output cluster_name)
 ```
 
-The
-[Kubernetes cluster name](https://github.com/hashicorp/learn-terraform-eks/blob/master/outputs.tf#L26)
-and [region](https://github.com/hashicorp/learn-terraform-eks/blob/master/outputs.tf#L21)
- correspond to the output variables showed after the successful Terraform run.
+The [Kubernetes cluster name](https://github.com/hashicorp/learn-terraform-eks/blob/master/outputs.tf#L26) and [region](https://github.com/hashicorp/learn-terraform-eks/blob/master/outputs.tf#L21) correspond to the output variables showed after the successful Terraform run.
 
 You can view these outputs again by running:
 
```

Comment:
We can delete all the content of the READ.me file from this line onwords as we are not deploying the metric-server and dashboard. In fact we will use other tools for logging and monitoring